---
format:
  html:
    embed-resources: true
---

# [GUI, tuantuan, PhD]{style="color: navy"}

School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai 200030, China

## [Contact Information]{style="color: #990000"}                                                       

- guituantuan\@sjtu.edu.cn                                                                           

- guituant2009\@163.com                                   

- Last updated on Dec-15-2024.                                                                       

## [Education]{style="color: #990000"}

#### - Shanghai Jiao Tong University, Shanghai, China (2020 - 2024)

-   **Ph.D candidate in Bioinformatics.**  School of Life Sciences and Biotechnology, supervised by Saijuan Chen, Zhu Chen & Hai Fang

#### - University of Chinese Academy of Sciences, Shanghai, China (2013 - 2016)

-   **Master in Bioinformatics.** Shanghai Institutes of Biological of Science, supervised by Yixue Li & Lei Liu

#### - Zhengzhou University, Zhengzhou, China (2009 - 2013)

-   **Bachelor in Bioengineering.**

## [Selected Publications]{style="color: #990000"}

**PhD period:**

1.  **Tuantuan Gui**, *et al.* **MNet: a metabolic network-centered tool for interpreting metabolomic and
transcriptomic data** (***Journal of Molecular Biology***, Under Review, First author)

2. Junyu Wang$^*$, **Tuantuan Gui**$^*$, *et al.* **Metabolomic insights into pathogenesis and therapeutic potential in
adult acute lymphoblastic leukemia** (***PNAS***, Revised, Co-first author)

**Master period:**

3. **Tuantuan Gui**, *et al.* **Identification of hepatocellular carcinoma-related genes with a machine learning and network analysis**, ***Journal Of Computational Biology***, 2015 Jan;22(1):63-71.

4. Zhi Liu, **Tuantuan Gui**, *et al.* **cisASE: a likelihood-based method for detecting putative cis-regulated allele-specific expression in RNA sequencing data**, ***Bioinformatics***, 2016 Nov;32(21):3291-3297. 

**Others:**

5. Min Emma Huang, Yining Qin, Yafang Shang, Qian Hao, Chuanzong Zhan, Chaoyang Lian, Simin Luo, Liu Daisy Liu, Senxin Zhang, Yu Zhang, Yang Wo, Niu Li, Shuheng Wu, **Tuantuan Gui**, Binbin Wang, Yifeng Luo, Yanni Cai, Xiaojing Liu, Ziye Xu, Pengfei Dai, Simiao Li, Liang Zhang, Junchao Dong, Jian Wang, Xiaoqi Zheng, Yingjie Xu, Yihua Sun, Wei Wu, Leng-Siew Yeap, Fei-Long Meng. **C-to-G editing generates double-strand breaks causing deletion, transversion and translocation**, ***Nature Cell Biology***, 2024 Feb;26(2):294-304.

6. Yanyan Wang, Senxin Zhang, Xinrui Yang, Joyce K Hwang, Chuanzong Zhan, Chaoyang Lian, Chong Wang, **Tuantuan Gui**, Binbin Wang, Xia Xie, Pengfei Dai, Lu Zhang, Ying Tian, Huizhi Zhang, Chong Han, Yanni Cai, Qian Hao, Xiaofei Ye, Xiaojing Liu, Jiaquan Liu, Zhiwei Cao, Shaohui Huang, Jie Song, Qiang Pan-Hammarström, Yaofeng Zhao, Frederick W Alt, Xiaoqi Zheng, Lin-Tai Da, Leng-Siew Yeap, Fei-Long Meng. **Mesoscale DNA feature in antibody-coding sequence facilitates somatic hypermutation**, ***Cell***, 2023 May;186(10):2193-2207.e19.

7. Qian Hao, Chuanzong Zhan, Chaoyang Lian, Simin Luo, Wenyi Cao, Binbin Wang, Xia Xie, Xiaofei Ye, **Tuantuan Gui**, Claudia Voena, Chiara Pighi, Yanyan Wang, Ying Tian, Xin Wang, Pengfei Dai, Yanni Cai, Xiaojing Liu, Shengqun Ouyang, Shiqi Sun, Qianwen Hu, Jun Liu, Youqiong Ye, Jingkun Zhao, Aiguo Lu, Ji-Yang Wang, Chuanxin Huang, Bing Su, Fei-Long Meng, Roberto Chiarle, Qiang Pan-Hammarström, Leng-Siew Yeap. **DNA repair mechanisms that promote insertion-deletion events during immunoglobulin gene diversification**, ***Science immunology***, 2023 Mar;8(81):eade1167.

8. Dingpeng Yang, Ying Sun, Jingjing Chen, Ying Zhang, Shuangshuang Fan, Min Huang, Xia Xie, Yanni Cai, Yafang Shang, **Tuantuan Gui**, Liming Sun, Jiazhi Hu, Junchao Dong, Leng-Siew Yeap, Xiaoming Wang, Wei Xiao, Fei-Long Meng. **REV7 is required for processing AID initiated DNA lesions in activated B cells**, ***Nature Communication***, 2020 Jun;11(1):2812.

## [Research Experience]{style="color: #990000"}

**Creation of R package MNet and website**

**Code:** https://github.com/tuantuangui/MNet    

1.	MNet has its built-in ‘dbMNet’ that provides an expert-curated knowledgebase for metabolomics, including all currently available gene-metabolite and metabolite-metabolite interactions in humans. 
3.	Its extended pathway analysis algorithms support extended pathway analysis, aiming to identify dysregulated metabolic pathways by considering both metabolites and genes. 
4.	Its network-driven capabilities for metabolism-related subnetwork analysis enable a systems-level understanding of metabolic interconnections.

**Analysis of the metabolism and RNA-seq data from acute lymphoblastic leukemia patients**  

1. The intracellular and serum metabolomic profile was constructed based on the MNet  
2. Enabling the development of a clustering system to supplement existing risk stratification based on the
comprehensive metabolic information dataset

![](overview.png)
**Figure 1.** Serum metabolome, cellular metabolome and cellular transriptome was analysised by software MNet. The results revealed a predominant increase in bile acid (BA) metabolites in serum, alongside metabolic rewiring that supported highly proliferative states and actively metabolic signaling, such as enriched nucleotide metabolism in leukemic blasts. By integrating intracellular metabolomics and transcriptomics, the Comprehensive Metabolic Information Dataset facilitated the development of a clustering system to supplement current risk stratification by MNet. The combined use of simvastatin with vincristine and dexamethasone regimen demonstrated a synergistic therapeutic effect in a murine ALL model by targeting the serum BA profile, effectively lowering key BA levels in serum and suppressing the infiltration of leukemic blasts in the liver.

## [Research Skills]{style="color: #990000"}

**Skills**  

**1. Development of New Methods:** I have developed a novel method to integrate the data from transcriptomics and metabolomics, which not only enables subnetwork analysis, but also facilitates extended pathway analysis.  

**2. Data Analyses:** I am proficient in clinical statistical analysis and multi-omics data analysis, including transcriptomics and metabolomics.  

**3. Liquid Biopsy Analysis:** I have developed a novel method to analyze circulating tumor DNA (ctDNA) , which could find the mutations or indels in cancer patients' plasma samples.

**4. Machine Learning:** I have applied a machine learning approach with maximum-relevance-minimum-redundancy (mRMR) algorithm followed by incremental feature selection (IFS) to a set of microarray data generated from hepatocellular carcinoma samples, and showed that the identified subnetwork is significantly enriched in biological processes related to cell death.

**5. Immunology Research:** I have applied the in-house pipeline to find the somatic hypermutation in IG genes, which was initiated by activation-induced cytidine deaminase (AID). This process generates mutations in the antibody-coding sequence, allowing for affinity maturation.

**6. Programming:** Proficient in using Linux operating systems, Perl and Python in data analysis, R in data analysis and visualization.

**Visualization**  

**1. R Shiny:** Proficient in R Shiny, with expertise in integrating various data sources and creating user-friendly interfaces.

**2. Adobe Illustrator:** Proficient in Adobe Illustrator, with a deep understanding of its vector drawing tools, typography features, and color management capabilities.   

## [Honors and Awards]{style="color: #990000"}
2016 Excellent Graduates in Shanghai  

2015 National Scholarship


